### PR TITLE
Ajout des zed launch files

### PR DESCRIPTION
**Pull request by nathan-gt**
for some reason les builds ne fonctionnent pas sur les pipeline, mais fonctionnent sur le jetson.

### CODE REVIEW

**Comment by patates-cipsi418** [comment]
Does the zed_wrapper come from the zed SDK?

**Comment by IliTheButterfly** [comment]
> Does the zed_wrapper come from the zed SDK?

Nope, it needs to be built on the side. On the Jetson it had to be built from source along a few other packages. However this could just be because we built ROS Humble from source. There are also dependencies with CUDA and such.

File: src/rove_zed/setup.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name + '/launch', glob('launch/*.launch.py')),
+        ('share/' + package_name + '/config', glob('config/*')),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='rove',
+    maintainer_email='capra@etsmtl.ca',
```
Comment (SimonR99) [inline src/rove_zed/setup.py:21]: bad email

**Review by SimonR99 — APPROVED** [review]
lgtm

**Pull request merged**